web/params.go

```diff
@@ -226,6 +230,22 @@ func ParamsFromRequest(r *http.Request) *Params {
 		params.LogsPerPage = val
 	}
 
+	if val, err := strconv.Atoi(query.Get("limit_after")); err != nil || val < 0 {
```

Comment:
Do you think it would make sense to just use a single `limit` parameter and have it apply to both before and after? I don't know how often someone would want to specify different values for `limit_before` and `limit_after`

Reply:
I am planning on using it in webapp because default payload when you have unreads is 120 posts and seems excessive.

My idea is to use like 20 above and probably 40-60 below because its good enough to have 20 or so above to provide the context of the conversation if needed.

I need to run this by ES before i can make definitely say if i am going to use it.

Reply:
Discussed with ST, proposal is to use 30 above, 30 below for now. But keep the variables separate given that we may tweak these during testing to get the right user experience.

Reply:
Alright, if that's what we're sure we want. I'd personally rather not add the new parameters if we don't need them since we won't be able to remove them in the future, but that's not a blocker